pandas/io/parquet.py

```diff
@@ -512,17 +511,21 @@ def read_parquet(
     DataFrame
     """
     impl = get_engine(engine)
-
-    use_nullable_dtypes = (
-        use_nullable_dtypes
-        if use_nullable_dtypes is not lib.no_default
-        else using_nullable_dtypes()
-    )
+    if use_nullable_dtypes is not lib.no_default:
+        warnings.warn(
+            "The argument 'use_nullable_dtypes' is deprecated. Use "
+            "dtype_backend='numpy_nullable' instead.",
```

Comment:
Nit: This message of using `dtype_backend='numpy_nullable'` isn't entirely correct if a user passes `use_nullable_dtypes=False` (for some reason). That bit can be added if `use_nullable_dtypes=True`

Reply:
Good catch. Updated